### PR TITLE
Create 2017-07-20-Jet.md

### PR DESCRIPTION
**Pull request by jbungo**

### CODE REVIEW

File: _posts/2017-07-20-Jet.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: 'Jet'
+date: 2017-07-20
+description:
```
Comment (tfoote) [inline _posts/2017-07-20-Jet.md]: You should add a one sentence descriptoin

**Comment by jbungo** [comment]
Done! Thanks again so much for the help.

Joe Bungo
Deep Learning Institute Program Manager
NVIDIA Corporation
developer.nvidia.com/teaching-kits<https://developer.nvidia.com/teaching-kits>
Mobile: +1 (512) 293-7324
jbungo@nvidia.com<mailto:jbungo@nvidia.com>

**Comment by jbungo** [comment]
Thumbnail image doesn’t display here for some reason: http://robots.ros.org/ - looks like its pointing correctly, are there restrictions on the thumbnail image?

Joe Bungo
Deep Learning Institute Program Manager
NVIDIA Corporation
developer.nvidia.com/teaching-kits<https://developer.nvidia.com/teaching-kits>
Mobile: +1 (512) 293-7324
jbungo@nvidia.com<mailto:jbungo@nvidia.com>

**Comment by tfoote** [comment]
The icon was not properly referenced. I fixed the images in https://github.com/ros-infrastructure/robots.ros.org/commit/f7629f79f4050021bbde3eedcbbeb8205a8cb0af

**Comment by jbungo** [comment]
Thanks a bunch!

Jet icon dips into the keywords at http://robots.ros.org/all/. Is that because description is to long, and/or too many tags?

Joe Bungo
Deep Learning Institute Program Manager
NVIDIA Corporation
developer.nvidia.com/teaching-kits<https://developer.nvidia.com/teaching-kits>
Mobile: +1 (512) 293-7324
jbungo@nvidia.com<mailto:jbungo@nvidia.com>

**Comment by tfoote** [comment]
Yes, the description is too long for the layout.

**Pull request merged**